src/main/scala/lms/thirdparty/cuda/cuda.scala

```diff
@@ -994,6 +1037,10 @@ trait CCodeGenCudaOps extends CCodeGenSizeTOps with CudaCodeGenLibFunction with
 
   override def traverse(n: Node): Unit = n match {
     case n @ Node(s, "NewSharedArray", xs, _) =>
+      // var tpe = typeMap.get(s)
+      // (1 to xs.length) foreach { _ => tpe = tpe.map(_.typeArguments.head) }
+      // val tpeAfter = remap(tpe.getOrElse(manifest[Unknown]))
+      // emit("__shared__ "); emit(s"$tpeAfter "); shallow(s);
```

Comment:
remove commented code if we don't need them any more

Reply:
fixed in b61827c